### DISCUSSION TITLE
CC1101 WOR

### DISCUSSION THREAD

**Discussion by frogg1ven** [jgromes/RadioLib#295] [General]
Hello, is there way to setup wake-up on radio with CC1101 module? Any example code for arduino to implement this function? Looking forward to hearing from you!

**Comment by jgromes** [comment]
Unfortuantely this is not currently supported by the high-level API. As far as I remember, out of the modules supported by RadioLib, only CC1101 has a WOR feature, which unfortuantely means it's not a high priority for me. Though I often accept PRs adding support for new stuff :)
